test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class UtilsTest {
+    
+    @Test
+    public void testNoNull() {   
```

Comment:
Note that we have a special naming convention for test method names. Please refer coding standard for more details
You can omit the `test` prefix in the name because we already have a `@Test` annotation.